styles/globals.css

```
@@ -580,6 +632,55 @@
 }
 
+/** Category glyph SVG — shared motion base (pair with group/cg + group-hover/cg:* on parts). */
+@utility cg-motion {
+  transform-box: fill-box;
+  transform-origin: center;
+
+  @media (prefers-reduced-motion: no-preference) {
+    transition:
+      transform 0.5s cubic-bezier(0.22, 1, 0.36, 1),
+      fill 0.5s ease,
+      opacity 0.5s ease;
```

Stylelint (17.12.0)

[error] Invalid position for declaration (no-invalid-position-declaration)

(no-invalid-position-declaration)